src/ProjectService.Broker/GetProjectIdsConsumer.cs

```diff
@@ -18,11 +18,6 @@ private object GetProjectIds(Guid userId)
         {
             var dbProjectUsers = _repository.Find(userId);
 
-            if (dbProjectUsers == null)
-            {
-                throw new NotFoundException($"User with id: {userId} was not found.");
-            }
```

Comment:
Why remove?

Reply:
Because If we don't have dbProjectUser we have to return an empty list, not throw exc. It is not error if somebody doesn't include in any project